src/Kernel.EFSupport/Helpers/ConnectionStringExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using LT.DigitalOffice.Kernel.Helpers;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Serilog;
+
+namespace LT.DigitalOffice.Kernel.EFSupport.Helpers
+{
+  public static class ConnectionStringExtension
```

Comment:
it is not connection string extension
rename to ConnectionStringHandler

Reply:
OK